scenes/game/local_game.gd

```diff
@@ -5438,7 +5798,10 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 				}
 				events += [create_event(Enums.EventType.EventType_CardFromHandToGauge_Choice, performing_player.my_id, min_amount, "", max_amount)]
 		"when_hit_force_for_armor":
-			performing_player.strike_stat_boosts.when_hit_force_for_armor = true
+			if 'use_gauge_instead' in effect and effect['use_gauge_instead']:
+				performing_player.strike_stat_boosts.when_hit_force_for_armor = "gauge"
```

Comment:
as mentioned earlier, maybe should have force take priority (don't set "gauge" here if it's already set to "force")